designs/administrator/4.5/alertmanager-config/index.md

```diff
@@ -0,0 +1,26 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Alertmanager Configuration
+
+![](img/alertmanager-first-time.png)
+- The default alertmanager configuration will ship with 3 receivers, two of which need to be configured to correctly function.
+
+![](img/alertmanager-editing-default.png)
+- The default receiver will continue to display an informational alert when it is configured explaining its purpose.
+
+![](img/alertmanager-incomplete-critical.png)
+- When only one receiver remains unconfigured, the text of the notification should change from plural to singular.
+
+![](img/alertmanager-editing-critical.png)
+- When the Critical receiver is edited, an informational alert explains the purpose of the receiver.
+- Once the receiver has been edited or deleted, this message no longer appears.
+
+![](img/alertmanager-all-configured.png)
+- Once all receivers except Watchdog are fully configured the notification is no longer necessary.
+
+![](img/alertmanager-editing-watchdog.png)
```

Comment:
Should Receiver Type be required here? Based on the fact that this receiver doesn't have to be configured and doesn't have a type initially it seems to me that it would be optional.

Reply:
Yeah, it's a little confusing.   The Watchdog receiver is purposefully left without a Receiver Type, so receiver will 'route' the watch dog alerts to nowhere, on purpose :-)

Reply:
Yeah, It ships in a "broken" state deliberately. The benefit of having that field be required is that if you start to edit this receiver the console will force you to either fully configure it or cancel out of the form which keeps you from ending up in a situation where you get a bunch of junk alerts

Reply:
Gotcha, thanks for the clarification!